packages/components/src/Toast/Toast.css

```diff
@@ -0,0 +1,50 @@
+:root {
+  --toast-shadow: 0 var(--space-base) var(--space-base) rgba(0, 0, 0, 0.16),
```

Comment:
Are we able to use `--shadow-base` from here here?

packages/design/src/shadows.css